integration/it-integration/src/main/java/se/inera/intyg/rehabstod/integration/it/service/IntygstjanstRestIntegrationServiceImpl.java

```diff
@@ -116,6 +118,15 @@ public SickLeavesResponseDTO getActiveSickLeaves(SickLeavesRequestDTO request) {
         return restTemplate.postForObject(url, request, SickLeavesResponseDTO.class);
     }
 
+    @Override
+    public PopulateFiltersResponseDTO getPopulatedFiltersForActiveSickLeaves(PopulateFiltersRequestDTO request) {
+        final String url = intygstjanstUrl + "/inera-certificate/internalapi/sickleave/populate/filters";
```

Comment:
A question that popped up when I saw your endpoint in SickLeaveController regardning my own implementation in IT, do you think it would be cleaner to change this endpoint to just /filters, instead of populate/filters?

Reply:
It might be cleaner with just /filters but both work.

Reply:
I suggest we go with /filters. I will make the needed changes in IT.